app/components/Home/Home.jsx

```diff
@@ -87,6 +103,8 @@ export default class Home extends React.PureComponent {
           </div>
         </section>
 
+        {videoDemo}
```

Comment:
Out of curiosity why not inlining ?

Reply:
Do you have an example of the best way ? I think the structure for different content to each language

Reply:
I think that what @steelstyle mean is that this would usually be done like:
```suggestion
        {locale === 'fr' && (
          <section>
            ...
          </section>
        )}
```

I personally think it's more readable this way (especially if you start having multiple conditions like that), but I have no blocker with your approach.

Reply:
ok, i'll do this approach